src/applications/mhv-medical-records/containers/LandingPage.jsx

```diff
@@ -286,24 +296,45 @@ const LandingPage = () => {
               <li>Height and weight</li>
               <li>Temperature</li>
             </ul>
-            <Link
-              to="/vitals"
-              className="vads-c-action-link--blue"
+            <va-link-action
+              type="secondary"
+              href="/my-health/medical-records/vitals"
               data-testid="vitals-landing-page-link"
-              onClick={() => {
+              text={VITALS_LABEL}
+              onClick={event => {
+                event.preventDefault();
+                history.push('/vitals');
                 sendAalViewList('Vitals');
                 sendDataDogAction(VITALS_LABEL);
                 logUniqueUserMetricsEvents(
                   EVENT_REGISTRY.MEDICAL_RECORDS_VITALS_ACCESSED,
                 );
               }}
-            >
-              {VITALS_LABEL}
-            </Link>
+            />
           </section>
 
           {!displayMarch17Updates && (
             <>
+              <section className="vads-u-padding-bottom--3">
+                <h2 className="vads-u-margin-top--4 vads-u-margin-bottom--1">
+                  What to do if you can’t find your medical records
+                </h2>
```

Comment:
This is hidden by the `displayMarch17Updates` flag, which is now on permanently, so this is not technically needed. However it won't hurt anything. We'll delete it when we get to this flag cleanup.